### PR TITLE
Change ExtensionVersion.filename to depend on pk, not version number (bug 1219789)

### PR DESCRIPTION
**Pull request by diox**
https://bugzilla.mozilla.org/show_bug.cgi?id=1219789

Slightly annoying to do this late, but it'd be much more painful to do this migration once we are live.

### CODE REVIEW

**Comment by ngokevin** [comment]
"to depend on pk"? in commit message

**Comment by diox** [comment]
oops

File: mkt/extensions/migrations/0020_change_filename_scheme.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import os.path
+
+from django.conf import settings
+from django.db import migrations
+
+from mkt.site.storage_utils import (move_stored_file, private_storage,
+                                    public_storage)
+
+
+def move_files_to_their_new_locations(apps, schema_editor):
```
Comment (ngokevin) [inline mkt/extensions/migrations/0020_change_filename_scheme.py]: how do we know this migration works? it's a lot of code

**Comment by ngokevin** [comment]
i'm worried about the migration, though testing it would involve a lot of mocking. but everything else looks good r+

File: mkt/extensions/migrations/0020_change_filename_scheme.py (hunk 1 above)
Comment (diox) [inline mkt/extensions/migrations/0020_change_filename_scheme.py]: That's a fair question. I tested it locally manually, but it's super annoying to test migrations (they work on frozen models etc). If you can test it locally (with some existing extensionversions) that'd be cool

**Pull request merged**